libr/core/cmd_write.c

```diff
@@ -534,6 +520,75 @@ static void cmd_write_value (RCore *core, const char *input) {
 	r_core_block_read (core);
 }
 
+static RCmdStatus common_wv_handler(RCore *core, int argc, const char **argv, int type) {
+	ut64 off = 0LL;
+	ut8 buf[sizeof(ut64)];
+	int wseek = r_config_get_i (core->config, "cfg.wseek");
+	bool be = r_config_get_i (core->config, "cfg.bigendian");
+
+	core->num->value = 0;
+	if (argc != 2) {
+		return R_CMD_STATUS_INVALID;
+	}
+
+	off = r_num_math (core->num, argv[1]);
+	if (core->file) {
+		r_io_use_fd (core->io, core->file->fd);
+	}
+
+	ut64 res = r_io_seek (core->io, core->offset, R_IO_SEEK_SET);
+	if (res == UT64_MAX) {
+		return R_CMD_STATUS_INVALID;
```

Comment:
also in this case, I guess, it is not exactly clear what's the meaning.

There should probably be something like:
COMMAND_NOT_FOUND,
WRONG_ARGS,
OK,
ERROR (error returned by the right command handler), like in this case.

Others?